media-gfx/iec16022/iec16022-0.2.7.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="2D barcode/DataMatrix generator"
+DESCRIPTION="With iec16022 you can produce 2D barcodes, also known as \
+DataMatrix, as defines in ISO/IEC 16022."
+HOMEPAGE="https://datenfreihafen.org/projects/iec16022.html"
+COPYRIGHT="2006 Stefan Schmidt"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/rdoeffinger/iec16022/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="34f00da5f6c83dd14b2803e5ed0f5310310e452023fe989c59fc5489b78cd33a"
+SOURCE_FILENAME="iec16022-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	iec16022$secondaryArchSuffix = $portVersion
+	lib:libiec16022$secondaryArchSuffix = 0.1.0 compat >= 0
+	cmd:iec16022$secondaryArchSuffix = $portVersion
```

Comment:
Should be
```
	cmd:iec16022 = $portVersion
```

Remember, we are using the commandBinDir "trick", so we should not expect to have the bin/iec16022-x86 symbolic link to bin/x86/iec16022 but just one runtime: bin/iec16022.